README.md

```diff
@@ -38,9 +38,9 @@ Instructions:
 3. If you have [nodejs](https://nodejs.org/en/), version 5.0+ is desired. If you don't, [install it](https://nodejs.org/en/download/). If you don't know, do `node --version` in your terminal/command prompt. If you don't get a number, you don't have it.
 4. In your command prompt/terminal, do `npm install`. This might take a few minutes. Now you'll be able to try out the current UI or play with the code!
 5. Do `npm start` in terminal to automatically open a new browser tab and see a local version of the site. Works best in Chrome.
-5a. npm bug workaround: please run `git checkout .` after any instances of running `npm install`. This is to circumvent a bug affecting the integrity of the package-lock.json file.
-6. Saved changes will automatically be reflected in-browser. If you don't see the changes, refresh the page. If nothing has changed, ask for help!
-7. Press cltr+c to stop the local server that you have running.
+6. npm bug workaround: please run `git checkout .` after any instances of running `npm install`. This is to circumvent a bug affecting the integrity of the package-lock.json file.
```

Comment:
For the future - try to keep PR's isolated to one feature/issue/bug/goal. It makes it smoother to pull stuff in. For example, if I like the rest of the code, but I don't like the way the README was changed, the infrastructure change that others may be waiting on gets held up by an unrelated change that wouldn't be conflicting with other changes.

For now, my personal take is that this note would be best just after the `npm install` step because it's immediately relevant. It's not a huge issue, though, not a reason to block the merge, and I'm happy to hear other thoughts!